common/src/model_registration/register.py

```diff
@@ -7,7 +7,7 @@
 import json
 import os
 import shutil
-from ruamel.yaml import YAML
+import yaml
```

Comment:
Sorry for the trouble... For consistency, it would be great to keep everything on ruamel.yaml. Is it just not present in the python-sdk-v2 environment?

Reply:
Yes @elesel. We thought to update components as compared to environment.